DataFormats/L1TCalorimeterPhase2/interface/CaloCrystalCluster.h

```diff
@@ -0,0 +1,152 @@
+#ifndef DataFormats_L1TCalorimeterPhase2_CaloCrystalsCluster_h
+#define DataFormats_L1TCalorimeterPhase2_CaloCrystalsCluster_h
+
+#include <vector>
+#include <map>
+#include <string>
+#include <algorithm>
+#include "DataFormats/L1Trigger/interface/L1Candidate.h"
+#include "DataFormats/DetId/interface/DetId.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+namespace l1tp2 {
+
+  class CaloCrystalCluster : public l1t::L1Candidate {
+  public:
+    CaloCrystalCluster()
+        : l1t::L1Candidate(),
+          calibratedPt_(0.),
+          hovere_(0.),
+          iso_(0.),
+          PUcorrPt_(0.),
```

Comment:
2.9 http://cms-sw.github.io/cms_coding_rules.html
`Start data member names with lower case. A trailing “_” is the preferred method to distinguish a data member from the getter method (e.g. momentum_).`